open-craft/edx-platform
# [BB-4218] Update Arabic translations (Juniper)

## Description

This PR adds the latest reviewed `Arabic` translation from Transifex for `opencraft-release/juniper.3`.

## Supporting information

- https://tasks.opencraft.com/browse/BB-4218

## Testing instructions

- Pull this PR in your local devstack
- Open LMS shell - `make lms-shell`
- Make `Arabic` default language by editing `/edx/etc/lms.yml`, set `LANGUAGE_CODE: ar`
- Check if updated strings working

## Deadline

None

## Other information

**Reviewer**
